extensions/dict-cc/src/components/TranslationsList.tsx

```diff
@@ -50,14 +42,28 @@ export default function Command() {
         });
       }
     }
+
     setLoading(false);
   };
 
+  useEffect(() => {
+    if (isSearchFromClipboard) {
+      (async () => {
+        const clipboardText = await Clipboard.readText();
+
+        if (clipboardText && clipboardText !== searchText) {
+          fetchTranslations(clipboardText);
+        }
+      })();
+    }
+  }, []);
```

Comment:
```suggestion
  }, [isSearchFromClipboard , fetchTranslations, searchText]);
```

PS: @pernielsentikaer, consider adding this ESLint rule to the extension template: https://www.npmjs.com/package/eslint-plugin-react-hooks.

Reply:
CC @thomaslombart